### PR TITLE
Update to go client beta 8

### PR DESCRIPTION
**Pull request by therve**

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
 PKG_NAME=datadog
 DIR=~/.terraform.d/plugins
 ZORKIAN_VERSION=master
-API_CLIENT_VERSION=master
```
Comment (bkabrda) [inline GNUmakefile:6]: We don't generally update this to contain the new release tag. The assumption is that when `make update-go-client` is run, you want to *update*, so fixing `API_CLIENT_VERSION` to the tag is not really useful.

Comment (therve) [reply GNUmakefile]: Yeah that was unwanted.

**Review by bkabrda — APPROVED** [review]

**Pull request merged**